docs/articles/faq/README.md

```diff
@@ -259,4 +262,27 @@ and [t.resizeWindow](../documentation/test-api/actions/resize-window.md#setting-
 to control the browser window size. If the test passes, it means your webpage hides
 the target element when the window is resized to smaller dimensions.
 
-Finally, try updating TestCafe to the latest version to see if the problem persists.
\ No newline at end of file
+Finally, try updating TestCafe to the latest version to see if the problem persists.
+
+### What are the possible reasons for an HTTP request failure? How can I address them?
+
+#### The Web server is not responding
+
+Check if the Web and DNS servers are online and configured to accept requests to this URL.
+
+#### Unstable or improperly configured network connection
+
+* Check the proxy server's, router's and network connection's settings.
+* Connect to a different network.
+* Restart or replace the router.
+* Use VPN.
+* Bypass or change the proxy server.
+
+#### Not enough resources in the container or CI system
```

Comment:
IMO, We described here what you can try to find a reason, not how to address this problem.

Reply:
Everything here is about "you could try", so I'd rephrase it.